### PR TITLE
Angular bootstrap 1.1.2

### PR DESCRIPTION
**Pull request by dgsmith2**
Updated angular-bootstrap to 1.1.2 in order to address #164.  Also fixed pre-existing broken unit tests

### CODE REVIEW

**Comment by niemyjski** [comment]
@dgsmith2 can you please bump the minor version. Last time we just incremented the revision on a major dependency upgrade people got pissed :( (was my fault).

**Comment by dgsmith2** [comment]
@niemyjski done!

**Comment by niemyjski** [comment]
@dgsmith2 do you think we should just bump everything to 1.5?

**Comment by dgsmith2** [comment]
I erred on the side of caution and went with 1.4 to mimic the version of Angular angular-bootstrap depends on.

I'm not 100% up to speed on the philosophy of specifying dependency versions, so forgive any naivety in the following question.  Should you not specify the earliest supported/compatible version?  By specifying 1.5 are you not saying it isn't supported on 1.4?

**Comment by niemyjski** [comment]
Yes, it's saying the recommended / requires 1.5, which means we only have to support the latest which is kind of nice and next week someone isn't asking us to bump the version again lol

**Comment by niemyjski** [comment]
@m-e-conroy thoughts?

**Comment by m-e-conroy** [comment]
Sorry, was away on vacation.  I'm all for bumping to 1.5 and only supporting the latest versions.

**Comment by niemyjski** [comment]
@dgsmith2 can you bump this up and I'll create a release

**Comment by dgsmith2** [comment]
Done!

File: example/index2.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<div class="row">
 			<div class="col-md-12">
 				<h1>Dialog Service Demo (v5.3) <span class="text-muted"><small>Sans Angular-Translate</small></span></h1>
-				<p class="text-info">(Using: Bootstrap 3.3.6, AngularJS 1.4.8, Angular UI Bootstrap 0.14.3)</p>
+				<p class="text-info">(Using: Bootstrap 3.3.6, AngularJS 1.4.9, Angular UI Bootstrap 1.1.2)</p>
 				<p>This Demo of Angular-Dialog-Service <strong>features an automatic substitution for Angular-Translate</strong>.  It will detect whether or not Angular-Translate is loaded; if not it will use the included translate substitute (translate.sub) module.  The translate substitution is small and modeled after Angular-Translate so that if in the future you wish to use Angular-Translate all you need to do is load the module (pascalprecht.translate) before this dialogs module.</p>
```
Comment (niemyjski) [inline example/index2.html]: should be using 1.5.x here

**Pull request merged**